solr/core/src/test/org/apache/solr/search/TestSearchPerf.java

```diff
@@ -173,7 +172,14 @@ private long doListGen(Query q, List<Query> filt) throws Exception {
     long ret = 0;
     for (int i = 0; i < ITERATIONS; i++) {
       DocList l =
-          searcher.getDocList(q, filt, (Sort) null, 0, 10, SolrIndexSearcher.NO_CHECK_QCACHE);
+          new QueryCommand()
+              .setQuery(q)
+              .setFilterList(filt)
+              .setOffset(0)
```

Comment:
```suggestion
```

Reply:
Hmm, I think the default is zero, no?

https://github.com/apache/solr/blob/releases/solr/9.6.1/solr/core/src/java/org/apache/solr/search/QueryCommand.java#L36

Reply:
(embarrassed) right of course